package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "publishr",
-  "version": "0.0.20",
+  "version": "0.0.22",
```

Comment:
Is this meant to be updated here? (I would assume we're only versioning on release per the scripts workflows...)